projects/vdk-plugins/vdk-server/setup.py

```diff
@@ -14,7 +14,13 @@
     description="Versatile Data Kit SDK plugin that facilitates the installation of a local Control Service.",
     long_description=pathlib.Path("README.md").read_text(),
     long_description_content_type="text/markdown",
-    install_requires=["vdk-core", "click-spinner", "docker", "kubernetes"],
+    install_requires=[
+        "vdk-core",
+        "click-spinner",
+        "docker",
+        "kubernetes",
+        "vdk-plugin-control-cli",
```

Comment:
Why are we adding the vdk-plugin-control-cli now?

Reply:
Because it was a missed dependency. vdk-server depends on vdk-control-cli for setting default control API URL. 
But the dependency was not added.